artifact/image/layerscanning/trace/testdata/populatelayers.yml

```diff
@@ -20,4 +20,4 @@ layers:
   - files:
       foo.txt:
         - foo
-        - foo
+        - foo2
```

Comment:
This got changed in a rework at some point last month, this is just changing it back.

Surprisingly it doesn't break any existing tests for a weird/complicated reason, but now with these tests it'll actually break if this is changed again, so that's good.